src/actions/Entities/VaccinePrescriptionActions.js

```diff
@@ -219,9 +219,9 @@ const createVaccinationNameNote = (
   UIDatabase.write(() => UIDatabase.create('NameNote', newNameNote));
 };
 
-const getPcdEventID = (patientEventType, patientId) => {
+const getPcdNameNoteID = patientId => {
   const getSelectedNameNote = UIDatabase.objects('NameNote')
```

Comment:
I'm unsure why this is called `getSelectedNameNote` when it isn't a method. Would you mind renaming to `selectedNameNotes` or something?

Reply:
Huh, yeah that's a bit confusing! b6323d5c0